R/zoom.R

```diff
@@ -67,10 +68,16 @@ get_zoomed_tbl <- function(dm) {
 #' @param new_tbl_name Name of the new table
 #'
 #' @export
-cdm_insert_zoomed_tbl <- function(dm, new_tbl_name) {
+cdm_insert_zoomed_tbl <- function(dm, new_tbl_name, repair = "check_unique") {
```

Comment:
Should we use `"unique"` as default here and for `cdm_add_tbl()` ? Can we allow `new_tbl_name = NULL` ? Do we also expose the `quiet` argument?

Reply:
Somehow I am not convinced by `"unique"` as default. But of course (together with accepting `new_tbl_name = NULL`) it would have the advantage, that `cdm_insert_zoomed_tbl()` would work when only providing the `dm` argument. Maybe discuss tomorrow?

no strong opinion about exposing `quiet`.

Reply:
Implemented it with `"unique"` as default and for cdm_add_tbl(). Allowed `new_tbl_name = NULL`. Exposed the `quiet` argument.